server/sql.go

```diff
@@ -463,13 +449,14 @@ func (s *Server) syncInstanceSchema(ctx context.Context, instance *api.Instance,
 		}
 		// Case 2, only appear in the synced db schema.
 		databaseCreate := &api.DatabaseCreate{
-			CreatorID:     api.SystemBotID,
-			ProjectID:     api.DefaultProjectID,
-			InstanceID:    instance.ID,
-			EnvironmentID: instance.EnvironmentID,
-			Name:          databaseName,
-			CharacterSet:  databaseMetadata.CharacterSet,
-			Collation:     databaseMetadata.Collation,
+			CreatorID:            api.SystemBotID,
+			ProjectID:            api.DefaultProjectID,
+			InstanceID:           instance.ID,
+			EnvironmentID:        instance.EnvironmentID,
+			Name:                 databaseName,
+			CharacterSet:         databaseMetadata.CharacterSet,
+			Collation:            databaseMetadata.Collation,
+			LastSuccessfulSyncTs: 0,
```

Comment:
Wondering why we pass 0 here (I guess that's also the reason we need to pass SyncTs to createDatabaseImpl)

Reply:
Yes, I'd like to sync database on-demand on zero lastSyncTs, in a follow-up PR.